### PR TITLE
Rename *Client classes that have business logic

### PR DESCRIPTION
**Pull request by rachaprince**
Rename the *Client classes that do more than simply interact with a service.

### CODE REVIEW

**Review by mannyjimenez0810 — APPROVED** [review]
No comments from Manny. Great Work!

File: firebase-app-distribution/src/main/java/com/google/firebase/app/distribution/CheckForNewReleaseHandler.java [hunk 1]
```diff
@@ -35,7 +35,11 @@
 import java.util.concurrent.Executor;
 import java.util.concurrent.Executors;
 
-class CheckForNewReleaseClient {
+/**
+ * Class that handles fetching the latest release from App Distribution and determining if it is a
+ * new release.
+ */
+class CheckForNewReleaseHandler {
```
[the remark this answers is not in the document]
Comment (rachaprince) [reply firebase-app-distribution/src/main/java/com/google/firebase/app/distribution/CheckForNewReleaseHandler.java]: I like it. Done

File: firebase-app-distribution/src/main/java/com/google/firebase/app/distribution/TesterSignInHandler.java [hunk 2]
```diff
@@ -42,7 +42,8 @@
 import com.google.firebase.installations.FirebaseInstallationsApi;
 import java.util.List;
 
-class TesterSignInClient {
+/** Class that handles signing in the tester */
+class TesterSignInHandler {
```
[the remark this answers is not in the document]
Comment (rachaprince) [reply firebase-app-distribution/src/main/java/com/google/firebase/app/distribution/TesterSignInHandler.java]: Agreed. My next PR will pull out the logic for showing the dialog, but we should consider more opportunistic refactors as they come up

[the remark this answers is not in the document]
Comment (rachaprince) [reply firebase-app-distribution/src/main/java/com/google/firebase/app/distribution/TesterSignInHandler.java]: Updated name to TesterSignInManager

**Review by lfkellogg — APPROVED** [review]
Looks great!

**Pull request merged**